source/backups.rst

```diff
@@ -112,8 +122,77 @@ for backups. Value of this key should be the same as the name used to
 create the secret object (``my-cluster-name-backup-s3`` in the last
 example).
 
-The schedule is specified in crontab format as explained in
-:ref:`operator.custom-resource-options`.
+.. _backups.scheduled-azure:
+
+Backups on Microsoft Azure Blob storage
+***************************************
+
+Since backups are stored separately on `Azure Blob Storage <https://azure.microsoft.com/en-us/services/storage/blobs/>`_,
+a secret with ``AZURE_STORAGE_ACCOUNT_NAME`` and ``AZURE_STORAGE_ACCOUNT_KEY`` should be present on
+the Kubernetes cluster. The secrets file with these base64-encoded keys should
+be created: for example ``deploy/backup-azure.yaml`` file with the following
+contents.
+
+
+.. code:: yaml
+
+   apiVersion: v1
+   kind: Secret
+   metadata:
+     name: my-cluster-azure-secret
+   type: Opaque
+   data:
+     AZURE_STORAGE_ACCOUNT_NAME: UkVQTEFDRS1XSVRILUFXUy1BQ0NFU1MtS0VZ
+     AZURE_STORAGE_ACCOUNT_KEY: UkVQTEFDRS1XSVRILUFXUy1TRUNSRVQtS0VZ
+
+.. note:: The following command can be used to get a base64-encoded string from
+   a plain text one: ``$ echo -n 'plain-text-string' | base64``
+
+The ``name`` value is the `Kubernetes
+secret <https://kubernetes.io/docs/concepts/configuration/secret/>`_
+name which will be used further, and ``AZURE_STORAGE_ACCOUNT_NAME`` and
+``AZURE_STORAGE_ACCOUNT_KEY`` credentials will be used to access the storage
+(and obviously they should contain proper values to make this access
+possible). To have effect secrets file should be applied with the appropriate
+command to create the secret object, e.g.
+``kubectl apply -f deploy/backup-azure.yaml`` (for Kubernetes).
+
+All the data needed to access the Azure Blob storage to store backups should be
+put into the ``backup.storages`` subsection, and ``backup.schedule`` subsection
+should actually schedule backups in crontab-compatible way. Here is an example
+of `deploy/cr.yaml <https://github.com/percona/percona-xtradb-cluster-operator/blob/main/deploy/cr.yaml>`__
+which uses Azure Blob storage for backups:
+
+.. code:: yaml
+
+   ...
+   backup:
+     enabled: true
+     ...
+     storages:
+       azure-blob:
+         type: azure
+         azure:
+           container: <your-container-name>
+           credentialsSecret: my-cluster-azure-secret
+
+     ...
+     schedule:
+      - name: "sat-night-backup"
+        schedule: "0 0 * * 6"
+        keep: 3
+        storageName: azure-blob
+     ...
+
+The options within these three subsections are further explained in the
+:ref:`Operator Custom Resource options<operator.backup-section>`.
```

Comment:
Can we at least explain `azure.container` option? I understand that other options are clear, but this one is relevant to Azure Blob Storage.